docs/projects.md

```diff
@@ -4,36 +4,37 @@ A project is an app that contains React code and has a dependency on `react-nati
 
 Projects can provide additional properties to alter the CLI behavior, such as custom location of React Native files (this is useful when running RNTester from source) or a non-standard location of a project (useful when working in a brownfield app).
 
-## How does it work?
-
-A project can define a `react-native.config.js` at the root with custom configuration to be picked up by the CLI.
-
-For example, below configuration informs CLI of the additional assets to link and about a custom project location.
-
-```js
-module.exports = {
-  project: {
-    ios: {
-      project: './CustomProject.xcodeproj',
-    },
-  },
-  assets: ['./assets'],
```

Comment:
I think it's time to figure out a better story for assets, which are not autolinked and currently require running `react-native link`, which is now removed

Reply:
I have already started appropriate discussions on the Discord. I am hoping to write a follow-up PR with proper, built-in support for assets (in case it doesn't work right now).